ci/dash/build_src.sh

```diff
@@ -63,7 +63,3 @@ fi
 if [ "$RUN_SECURITY_TESTS" = "true" ]; then
   make test-security-check
 fi
-
-if [ "$RUN_SYMBOL_TESTS" = "true" ]; then
```

Comment:
Do we have guix builds for each PR? If we do that, we will find a potential issue during guix build in pre-release time, which can make mess and release delay.

I checked logs of some guix guild here https://github.com/dashpay/dash/actions/runs/11645989967/job/32429737929?pr=6379#step:20:1 and I don't see `check-symbols` running. Does it?